src/System.CommandLine.Hosting/InvocationLifetime.cs

```diff
@@ -7,6 +7,11 @@
 using Microsoft.Extensions.Logging.Abstractions;
 using Microsoft.Extensions.Options;
 
+#if NETSTANDARD2_0
+using IHostEnvironment = Microsoft.Extensions.Hosting.IHostingEnvironment;
+using IHostApplicationLifetime = Microsoft.Extensions.Hosting.IApplicationLifetime;
+#endif
+
```

Comment:
The types listed here are deprecated in `netstandard2.1`. But the new types do not exist in `netstandard2.0`. Using these conditional using-aliasing statements ensures that we use the new types from now on.